handsontable/src/core.js

```diff
@@ -138,7 +136,23 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
    */
   this.executionSuspendedCounter = 0;
 
-  keyStateStartObserving(this.rootDocument);
+  const shortcutManager = createShortcutManager({
+    beforeKeyDown: (event) => {
+      if (this.isListening() === false) { // Performing action (executing a callback) and triggering hook only for listening Handsontable.
+        return false;
+      }
+
+      return this.runHooks('beforeKeyDown', event);
+    },
+    afterKeyDown: (event) => {
+      if (this.isDestroyed) { // Handsontable could be destroyed after performing action (executing a callback).
+        return;
+      }
+
+      this.runHooks('afterDocumentKeyDown', event);
+    },
```

Comment:
WDYT to introduce a new hook `afterKeyDown`? Seems to be reasonable to unify the `keyDown` hook names. The `afterDocumentKeyDown` hook would stay for backward compatibility support.
```suggestion
    beforeKeyDown: (event) => {
      if (this.isListening() === false) { // Performing action (executing a callback) and triggering hook only for listening Handsontable.
        return false;
      }

      return this.runHooks('beforeKeyDown', event);
    },
    afterKeyDown: (event) => {
      if (this.isDestroyed) { // Handsontable could be destroyed after performing action (executing a callback).
        return;
      }

      this.runHooks('afterKeyDown', event);
      this.runHooks('afterDocumentKeyDown', event);
    },
```

Reply:
I'm against increasing the scope with yet another change. The hooks are not in scope of the SM or A11Y project.